lib/routes/bflai/announcements.ts

```diff
@@ -0,0 +1,193 @@
+import { Route, DataItem, Data } from '@/types'; // RSSHub 类型定义
```

Comment:
Please use [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) `bfl` instead of `bflai`.